src/WSCryptDecryptSoapClient.php

```diff
@@ -83,9 +82,10 @@ private function getStreamContext($testEnv = false, $caFile = null)
             $host = $this->certificatePeerName['production'];
         }
 
-        $this->streamContextOption['ssl']['crypto_method'] = STREAM_CRYPTO_METHOD_TLS_CLIENT;
-        $this->streamContextOption['ssl']['verify_peer'] = true;
-        $this->streamContextOption['ssl']['SNI_enabled'] = true;
+        $this->streamContextOption['ssl']['crypto_method'] = STREAM_CRYPTO_METHOD_ANY_CLIENT;
```

Comment:
Using any client permit lowering crypto method down to sslv2, Gestpay is moving everything to tlsv1.2
(also STREAM_CRYPTO_METHOD_ANY_CLIENT is availably from php 5.6, library minimum requirement is php 5.3.3)